platform/testing/contract-testing/how-to-use-pact-for-contract-testing.md

```diff
@@ -77,76 +74,84 @@ VFS teams are required to use Pact for contract testing in the following scenari
+PRs related to Pacts will go through the standard [code review process](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/code_review_guidelines.md).
 
-### Process
+### Workflow
 
-1. `vets-website`  runs unit tests to validate its request and response interactions with `vets-api` endpoints.
-2. `vets-website` generates contracts (referred to as **pacts** in the Pact framework) 
-3. Pacts publish to a central location/broker ([Heroku](https://vagov-pact-broker.herokuapp.com/)) to be versioned and shared with the provider (`vets-api`).
-4. The provider (`vets-api`) runs a rake task to verify the pacts. This verification task replays the requests defined in the pacts against the provider (`vets-api`) and validates that the actual response matches the expected response.
-5. Results of the verification are published back to the broker.
+The process is a collaborative effort where front-end (FE) and back-end (BE) engineers should communicate and may need to iterate on the pacts to get them successfully verified.
 
-Builds depend on successful verification results to deploy.
+#### Front-end workflow
 
-*Note: In future iterations of the VSP Pact Process, we plan to integrate the rake verification task into CircleCI as part of the build step.* 
+1. FE engineers write Pact tests in feature branches in vets-website.
+2. The vets-website CI pipeline runs the contract tests job.
+   - The job runs all Pact tests, which generate pacts if they pass.
+   - The pacts are then published to a central broker.
+3. The vets-website CI pipeline runs a job to check `can-i-deploy`.
+   - The `can-i-deploy` task returns the verification statuses of all pacts against vets-api's master branch.
+     This determines whether it's safe to merge this vets-website feature branch.
+   - If there is no status yet for a pact, it will poll for 30 seconds while the pacts are asynchronously verified.
+   - If the status of any pact can't be determined, the job times out with a failure.
+4. The broker triggers the verification job in the vets-api CI pipeline, which publishes its results to the broker.
+5. If all pacts in the branch are successfully verified, the branch can be merged.
+6. If verification has failed for any pact, FE engineers should discuss with BE engineers to resolve the failure.
+   **The branch should not be merged until the `can-i-deploy` task is successful.**
+   FE should adjust the pact or BE should update provider states or API responses to accommodate the pact.
+   If the resolution only involves BE changes, wait for those changes to get merged and re-run the vets-website CI pipeline.
 
+#### Back-end workflow
 
-![](https://i.imgur.com/zQMyDS0.png)
+1. BE engineers update API endpoints and/or provider states in a feature branch in vets-api.
+2. The vets-api CI pipeline runs the verification job, which verifies the all of the latest pacts on the vets-website master branch.
```

Comment:
typo here: "the all of the"

Reply:
Maybe add something about how you can view the pact verification results in CircleCI

Reply:
I intended for this section to be more conceptual, but I added information about viewing the verification results in CircleCI in another section.